### PR TITLE
fix fragile csv finding in KPM/OCRB viewss

### LINKED ISSUE #23: Fix sloppy relative import in views.py
Example line:
`        f = '../Data/Budget_in_Brief_OCRB_data_All_Years.csv'
`

Should be using sys.path and the filepath to to find the csv, not just a string.

### PR DESCRIPTION
**Pull request by hassanshamim**
closes #23 

Added another constant to the django settings with proper location of the data directory.  Access it with `from django.conf import settings     settings.BASE_DATA_DIR`

Also moved the csv parsing logic into their own functions, so we can use the generics.Listview interface the same way we will when we get data in the db.

### CODE REVIEW

**Comment by hassanshamim** [comment]
viewsss 🐍

File: budget_proj/budget_app/views.py [hunk 1]
```diff
@@ -22,59 +23,58 @@
...
+    all_objects = [models.KPM(**row) for row in reader]
+    for obj in all_objects: # remove empty strings from number fields, fix in serializer?
+        if obj.amount == '':
+            obj.amount = None
+    return all_objects
+
 class ListOcrb(generics.ListAPIView):
     """
     A class based view that inherits from the generics class. The generics
     class gives you a convinient way to declare views quickly when you only
```
Comment (jimtyhurst) [inline budget_proj/budget_app/views.py]: typo: convinient => convenient

**Comment by jimtyhurst** [comment]
I like that the CSV parsing logic has been refactored into separate functions.

**Comment by MikeTheCanuck** [comment]
I hacked these changes into a separate branch to enable my work to Dockerize this repo, and (with a couple of minor tweaks) this worked like a charm.

Seems like a solid addition to our codebase.

**Pull request merged**